docs/source/conf.py

```diff
@@ -28,16 +28,20 @@
     )
 )
 
+import libertem
+
 # -- Project information -----------------------------------------------------
 
 project = 'LiberTEM'
 copyright = '2018, LiberTEM Authors'
 author = 'LiberTEM Authors'
 
+_version_bits = libertem.__version__.split('.')
 # The short X.Y version
-version = ''
+version = _version_bits[0] + '.' + _version_bits[1]
```

Comment:
Hm, what about the patch/fix part of `__version__`? Should they not be part of `version`?

Reply:
Conf has two variables, version and release. Version is supposed to be "X.Y" according to the inline documentation in conf.py.

Reply:
`release` is the full `libertem.__version__`

Reply:
To be honest, I still didn't find a place where these variables actually show up in the documentation.

Reply:
Ok, looks good, we will still need to adapt the template to include the version in the footer etc.